src/main/java/org/altbeacon/beacon/service/RangedBeacon.java

```diff
@@ -49,65 +56,26 @@ public void commitMeasurements() {
         }
     }
 
-	public static void setSampleExpirationMilliseconds(long milliseconds) {
```

Comment:
I see... changed this, too. But need to store the value of "sampleExpirationMilliseconds" in RangedBeacon (because its set in a static method) and transfer it to the filter on instantiation